docs/source/installation.rst

```diff
@@ -1,2 +1,54 @@
+Requirements
```

Comment:
You can create a new file requirements.rst. Also, the dependencies is handled by poetry and described in pyproject.toml. Perhaps it is not necessary